src/components/LoadingSpinner/index.ts

```diff
@@ -1,3 +1,4 @@
 import dynamic from 'next/dynamic'
 
 export const LoadingSpinner = dynamic(() => import('./LoadingSpinner'), { ssr: false })
```

Comment:
I think that this is no longer necessary.

Reply:
I don't get this point, could you explain further? the way I understand it, this is using barrel import/export

Reply:
well we dont need the dynamic import anymore since its a regular component, isnt it?